corehq/apps/users/models.py

```diff
@@ -2256,6 +2256,14 @@ def get_dimagi_emails_by_domain(cls, domain):
             if user_doc['email'].endswith('@dimagi.com'):
                 yield user_doc['email']
 
+    def get_location_id(self, domain):
+        return getattr(self.get_domain_membership(domain), 'location_id', None)
+
+    def get_location(self, domain):
```

Comment:
minor: this gets called quite a bit, may be a good case for memoize or quickcache